src/engine.ts

```diff
@@ -393,6 +395,65 @@ export class Engine {
     }
   }
 
+  /**
+   * Include instruction.
+   * 
+   * inst[1]  - name of macro or partial to include
+   * inst[2]  - optional arguments
+   */
+  executeInclude(inst: IncludeCode, ctx: Context): void {
+    // Refuse to evaluate the instruction if not explicitly enabled.
+    if (!ctx.enableInclude) {
+      return;
+    }
+
+    const name = inst[1];
+    const rawargs = inst[2];
+
+    // Fetch the partial or macro code
+    const code = ctx.getPartial(name);
+    if (!Array.isArray(code)) {
+      ctx.error(partialMissing(name));
+      return;
+    }
+
+    // Set options from arguments
+    let output = false;
+    if (Array.isArray(rawargs)) {
+      for (const arg of rawargs[0]) {
```

Comment:
Would `Array.includes('output')` be less wordy?

Reply:
I don't believe `Array.includes` is supported on IE, but otherwise yes.